defaults/main.yml

```diff
@@ -0,0 +1,103 @@
+---
+
+# Install set of standard packages
+rsyslog_packages: [ 'rsyslog', 'logrotate' ]
+
+# Should rsyslog role manage server pools?
+rsyslog_manage_pools: True
+
+# List of managed pools
+rsyslog_pools: [ '{{ rsyslog_pool_default }}', '{{ rsyslog_pool_cron }}' ]
+
+# Default rsyslog pool , explanation of variables can be found
+# in templates/etc/rsyslog.d/pool.conf.j2
+# Everything commented out is optional.
+rsyslog_pool_default:
+  enabled: True
+  name: '50-default'
+  options: |
+    #  Default rules for rsyslog.
+    #
+    #>-->--->---For more information see rsyslog.conf(5) and /etc/rsyslog.conf
+
+    #
+    # First some standard log files.  Log by facility.
+    #
+    auth,authpriv.*>>--->---/var/log/auth.log
+    *.*;auth,authpriv.none>->----/var/log/syslog
+    #cron.*>>--->--->---/var/log/cron.log
+    #daemon.*>-->--->----/var/log/daemon.log
+    kern.*>->--->--->----/var/log/kern.log
+    #lpr.*>->--->--->----/var/log/lpr.log
+    mail.*>->--->--->----/var/log/mail.log
+    #user.*>>--->--->----/var/log/user.log
+
+    #
+    # Logging for the mail system.  Split it up so that
+    # it is easy to write scripts to parse these files.
+    #
+    #mail.info>->--->----/var/log/mail.info
+    #mail.warn>->--->----/var/log/mail.warn
+    mail.err>--->--->---/var/log/mail.err
+
+    #
+    # Logging for INN news system.
+    #
+    news.crit>-->--->---/var/log/news/news.crit
+    news.err>--->--->---/var/log/news/news.err
+    news.notice>>--->----/var/log/news/news.notice
```

Comment:
@le9i0nx Just a heads-up - could you remove the `>---` characters from the pull request? These represent Tabs and I think they somehow got mixed in your commits, but with that in the config file `rsyslog` won't work.